lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -17,11 +17,16 @@ def __init__(self, parent_axes, aux_transform=None,
         kwargs["frameon"] = False
         super().__init__(parent_axes.figure, parent_axes._position, **kwargs)
 
-    def cla(self):
-        super().cla()
+    def clear(self):
+        # docstring inherited
+        super().clear()
         martist.setp(self.get_children(), visible=False)
         self._get_lines = self._parent_axes._get_lines
 
+    def cla(self):
+        """Clear the current axes."""
+        self.clear()
```

Comment:
```suggestion
```
This is an interesting one because it looks like this isn't inheriting from Axes, but down below there is a class factory to create the ParasiteAxes from Axes, which I think means this does actually get a `cla()` from Axes when used in practice.

Reply:
Removing the `def cla(self)` because `ParasiteAxes` inherits from Axes with the class factory, as seen on lines 97-99 of `lib/mpl_toolkits/axes_grid1/parasite_axes.py`.